Manifest.toml

```diff
@@ -1,8 +1,8 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.9.0"
+julia_version = "1.10.4"
```

Comment:
This is currently not possible. We have to use Julia 1.9.X at the moment, because if we upgrade then many features in the OrbitTomography.jl package cannot be used. Specifically the PolyharmonicSplines.jl package has some functions that cannot be used in Julia 1.10. This package is used by the OrbitTomography.jl package. I am currently working on a fix for this. But until we have this fix in place, we cannot use Julia version 1.10 for the OWCF, since then the OrbitTomography.jl package would stop working.